tests/kernel/bpftrace.pm

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Package: bpftrace
+# Summary: Compile and attach eBPF probes with bpftrace
+# Maintainer: kernel-qa@suse.de
+
+use Mojo::Base qw(opensusebasetest);
+use testapi;
+use utils 'zypper_call';
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+
+    zypper_call('in bpftrace bpftrace-tools');
+
+    my $tools_dir = '/usr/share/bpftrace/tools';
+    my @tests = qw(
+      bashreadline.bt
+      biolatency.bt
+      biosnoop.bt
+      biostacks.bt
+      bitesize.bt
+      capable.bt
+      cpuwalk.bt
+      dcsnoop.bt
+      execsnoop.bt
+      gethostlatency.bt
+      killsnoop.bt
+      loads.bt
+      mdflush.bt
+      naptime.bt
+      oomkill.bt
+      opensnoop.bt
+      pidpersec.bt
+      runqlat.bt
+      runqlen.bt
+      setuids.bt
+      ssllatency.bt
+      sslsnoop.bt
+      statsnoop.bt
+      swapin.bt
+      syncsnoop.bt
+      syscount.bt
+      tcpaccept.bt
+      tcpconnect.bt
+      tcpdrop.bt
+      tcplife.bt
+      tcpretrans.bt
+      tcpsynbl.bt
+      threadsnoop.bt
+      undump.bt
+      vfscount.bt
+      vfsstat.bt
+      writeback.bt
+      xfsdist.bt
+    );
+
+    foreach my $t (@tests) {
+        script_run("timeout --preserve-status -s SIGINT 5 bpftrace $tools_dir/$t");
```

Comment:
It will just run command without any return value check or  comparison with expected value. It will run, but if something fails, we will not spot it (i think tests will be all the time green).  Could you you please add one or two tests which will compare script output with some expected result?